python/py-nbxmpp/Portfile

```diff
@@ -1,35 +1,39 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem          1.0
-PortGroup           python 1.0
+PortSystem           1.0
+PortGroup            python 1.0
+PortGroup            gitlab 1.0
 
-name                py-nbxmpp
-version             2.0.6
-revision            0
-categories-append   net
-platforms           darwin
-license             GPL-3
-supported_archs     noarch
+gitlab.instance      https://dev.gajim.org
+gitlab.setup         gajim python-nbxmpp 3.0.1 nbxmpp-
 
-python.versions     27 37 38 39 310
+name                 py-nbxmpp
+version              3.0.1
+revision             0
+categories-append    net chat
+license              GPL-3
+supported_archs      noarch
+maintainers          nomaintainer
 
-maintainers         nomaintainer
+description          Non blocking Jabber/XMPP module
 
-description         Non blocking Jabber/XMPP module
+long_description     The goal of this python library is to provide a way \
+                     for python applications to use Jabber/XMPP networks in a non-blocking \
+                     way. Originally, a fork of the xmpppy jabber python library.
 
-long_description    The goal of this python library is to provide a way for \
-                    python applications to use Jabber/XMPP networks in a non-blocking \
-                    way. Originally, a fork of the xmpppy jabber python library.
+checksums    rmd160  069352f7ead2dd75a9c1ca6a9373572afeea4833 \
+             sha256  ad5fca43c902d7ae2317a628062eda08ec0c1ebbbd3066e43cfbd536fa6fef64 \
+               size  126138
 
-homepage            https://dev.gajim.org/gajim/python-nbxmpp
-master_sites        pypi:n/nbxmpp/
-
-distname            nbxmpp-${version}
-
-checksums           rmd160  979526910665550bc7e2285c4833a466182b07fd \
-                    sha256  0fc486894e546a56d8b2baa587787aa3cc9aa3cb76912c25598c7fd11db848bd \
-                    size    141999
+python.versions      39 310
 
 if {${name} ne ${subport}} {
-    livecheck.type  none
+   depends_build-append port:py${python.version}-setuptools
+   depends_lib-append   port:glib2 \
```

Comment:
this should probably be `path:lib/pkgconfig/glib-2.0.pc:glib2` as in pretty much every other ports, so that the `-devel` port can also satisfy this depdency